### PR TITLE
Added more preview cards

### PR DESCRIPTION
**Pull request by Chr1Z93**
- added Ancestral Token
- added Fine Tuning
- added Key of Solomon
- added Purified
- added Drain Essence
- added Fake Credentials
- added Prismatic Spectacles
- added Kate + signatures
- added Transmogrify
- added Well-Dressed

### CODE REVIEW

**Comment by Entrox-Licher** [comment]
Do you wanna go ahead and add slot metadata for these now that we have that?

**Comment by Chr1Z93** [comment]
> Do you wanna go ahead and add slot metadata for these now that we have that?

Done

**Comment by Entrox-Licher** [comment]
Nitpick: Have the slot field after `type` instead of after `trait` for consistency

**Review by Entrox-Licher — CHANGES_REQUESTED** [review]
Minor change for Kate, other than that it looks good to go!

File: objects/AllPlayerCards.15bb07/KateWinthrop.ce2322.gmnotes [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+  "id": "10004",
+  "type": "Investigator",
+  "class": "Seeker",
+  "traits": "Miskatonic. Scholar.",
+    "bonded": [
```
[the remark this answers is not in the document]
Comment (Chr1Z93) [reply objects/AllPlayerCards.15bb07/KateWinthrop.ce2322.gmnotes:6]: Good catch. Corrected it!

**Review by Entrox-Licher — APPROVED** [review]

**Pull request merged**